### ISSUE TITLE
Use async/await in examples

### ISSUE THREAD

**Issue by kenchris** [jtguggedal/thingy_web_bluetooth#5]
And don't use var today, use const by default or let when reassignment is needed:

ie:

```
async function start() {
  try {
    const thingy = new Thingy();
    await thingy.connect();
    const name = await thingy.nameGet();
    console.log(`This Thingy:52 is called ${name}`;
    thingy.ledSetBreathe(5, 20, 1500); 
  catch(err) {
    console.error(err); 
  }
}
```

or

```
async function start() {
  const thingy = new Thingy();
  await thingy.connect();
  const name = await thingy.nameGet();
  console.log(`This Thingy:52 is called ${name}`;
  thingy.ledSetBreathe(5, 20, 1500); 
}

try {
  start();
} catch(err) {
  console.error(err); 
}
```

**Comment by jtguggedal** [comment]
Thanks for your feedback in the issues you posted!
And sorry for the late response, for some reason I haven't received notifications. 

I've finally found some time to work more on this project, an have started to implement it as an ES6 class and using async/await. Work in progress [in this branch](https://github.com/jtguggedal/thingy_web_bluetooth/tree/async). 

I have updated the example per your suggestion.
